securitycenter/snippets_v2/snippets_notification_configs_v2.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+#
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Demos for working with notification configs."""
+
+
+# [START securitycenter_create_notification_config_v2]
+def create_notification_config(parent_id, location_id, pubsub_topic, notification_config_id):
```

Comment:
Minor issues: Please update the docs strings

Example: You can `function_with_types_in_docstring` in https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

1. Add one line description of the function
2. Use type hints
3. Add missing return types

Reply:
addressed